tasty-plutus/src/Test/Tasty/Plutus/Context.hs

```diff
@@ -193,6 +195,21 @@ data Output
       Show
     )
 
+{-- | A valid time range for a transaction. -}
+newtype ValidRange = ValidRange POSIXTimeRange
```

Comment:
Needs `@since` annotations, [as per the style guide](https://github.com/mlabs-haskell/styleguide/blob/main/TLDR.md#documentation).